gsa/src/web/pages/policies/component.js

```diff
@@ -768,12 +766,13 @@ class PolicyComponent extends React.Component {
         )}
         {editNvtDetailsDialogVisible && (
           <EditNvtDetailsDialog
-            policy={policy}
-            policyName={policyName}
+            config={policy}
+            config_name={policyName}
             family_name={family_name}
             manual_timeout={manual_timeout}
             nvt={nvt}
             preference_values={preference_values}
+            text={_('Policy')}
```

Comment:
Same as with the the text param in the previous commit. This time you are requiring the text already, though.